piqueserver/core_commands/social.py

```diff
@@ -88,7 +88,8 @@ def to_admin(connection, *arg):
             prefix = '\x0304' + prefix + '\x0f'
         irc_relay.send(prefix + ' <%s> %s' % (connection.name, message))
     for player in protocol.players.values():
-        if player.admin and player is not connection:
+        if (player.admin or player.user_types.moderator or
+                    player.user_types.guard) and player is not connection:
```

Comment:
This would be more readable as a variable:

```suggestion
        can_view_admin_messages = (
            player.admin or
            player.user_types.moderator or
            player.user_types.guard
        )

        if can_view_admin_messages and player is not connection:
```